OpenRA.Mods.Common/Traits/Air/Aircraft.cs

```diff
@@ -125,6 +125,9 @@ IEnumerable<object> IActorPreviewInitInfo.ActorPreviewInits(ActorInfo ai, ActorP
 			"This only applies while the aircraft is above terrain which is listed in LandableTerrainTypes.")]
 		public readonly BooleanExpression LandOnCondition;
 
+		[Desc("Type of exit used for landing at building.")]
+		public readonly string LandingExitType = null;
```

Comment:
You could probably then leave this out and just hardcode `null` to keep the previously behaviour.  @forcecore's docking PRs will eventually solve the landing points problems in a better way.

Reply:
Well it used (LandingExitType) just twice in entire code base so changing both to null wouldn't be an issue. However wouldn't it impact readability of the code somewhat?

I mean, right now we all know what it is, even if it is essentially useless and temporary. But if we get rid of it we will get, at best, barely measurable performance improvement.

If you really want it gone, no problem, but it seems like unnecessary change. Especially if it is already being worked on by someone else.

Reply:
The issue is that if people start to use `LandingExitType` then we will need to worry about breaking their mods when it is eventually removed again.

Reply:
I have removed the variable and replaced all references to it with null. Currently I'm testing the code, will push after I'm done.